src/argparse/ArgumentParser.java

```diff
@@ -149,14 +141,11 @@ public ArrayList<Argument> getArguments() {
      */
     public void parseArguments() {
         helpArg.resolveArgument(args);
-        if (helpArg.isPassed()) {
-            printHelp();
-        }
+        if (helpArg.isPassed()) printHelp();
+        
         for (Argument arg : arguments) {
-            boolean success = arg.resolveArgument(args);
-            if (!success) {
-                System.out.println(String.format("Argument \"%s\" not used, or used incorrectly. See usage.\n",
-                        arg.getToken()));
+            if (!arg.resolveArgument(args)) {
```

Comment:
I kinda preffer this split up. It makes it a little more readable in my opinion. Additionally, I kinda want to keep line length < 120. Can you revert this line and line below.